modules/artifact-activation/src/test/java/org/mule/runtime/module/artifact/activation/internal/deployable/LightweightDeployableProjectModelBuilderTestCase.java

```diff
@@ -89,6 +99,34 @@ public void createDeployableProjectModelWithAdditionalDependenciesInAPlugin() th
     assertThat(descriptor.getArtifactId(), is("derby"));
   }
 
+
+  @Test
+  @Issue("W-14998254")
+  @Description("When pom has invalid GAV and no parent pom, then MuleRuntimeException will be the thrown.")
+  public void createDeployableProjectModelWithInvalidGAVAndMissingParentPom() throws Exception {
+    expectedException.expect(MuleRuntimeException.class);
```

Comment:
Add a condition involving the message of the `expectedException` to make sure we are not getting some other unrelated `MuleRuntimeException`, eventually making false assumptions about our coverage.